scala/scala.bzl

```diff
@@ -251,18 +251,6 @@ scala_repl = rule(
 
 _SCALA_BUILD_FILE = """
 # scala.BUILD
-java_import(
-    name = "scala-xml",
-    jars = ["lib/scala-xml_2.11-1.0.5.jar"],
-    visibility = ["//visibility:public"],
-)
-
-java_import(
-    name = "scala-parser-combinators",
-    jars = ["lib/scala-parser-combinators_2.11-1.0.4.jar"],
-    visibility = ["//visibility:public"],
-)
-
 java_import(
```

Comment:
can we kill the whole use of the sdk and just use jars? I think that might be the way to go so we can just use maven servers for all dependencies (or git repos I guess).

Reply:
Don't see why we couldn't, I'll update the PR after lunch